packages/icons/src/paths/Others.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import createSvgIcon from '../utils/createSvgIcon';
+
+export const Others = createSvgIcon(
+  <path
+    clip-rule="evenodd"
```

Comment:
you'll get console error for this clip-rule to clipRule and fill-rule to fillRule.

Reply:
Done